src/assignment/views/AssignmentOverview.tsx

```diff
@@ -215,20 +207,6 @@ const AssignmentOverview: FunctionComponent<
 		localStorage.setItem(ASSIGNMENT_OVERVIEW_BACK_BUTTON_FILTERS, JSON.stringify(query));
 	}, [query]);
 
-	useEffect(() => {
-		if (query.filter) {
-			setFilterString(query.filter);
-		}
-		if (query.sort_column) {
-			setSortColumn(query.sort_column as AssignmentOverviewTableColumns);
-		}
-		if (query.sort_order) {
-			setSortOrder(query.sort_order as Avo.Search.OrderDirection);
-		}
-		// Copy the query param value from the url once time when the page loads
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, []);
-
```

Comment:
`react-query` handles this now I assume?

Reply:
no, we just don't save them locally in a useState anymore
but directly in the url, so we never need to sync between url and useStates